components/usage/pkg/apiv1/usage.go

```diff
@@ -190,11 +190,20 @@ func (s *UsageService) GetCostCenter(ctx context.Context, in *v1.GetCostCenterRe
 }
 
 func dbCostCenterToAPI(c db.CostCenter) *v1.CostCenter {
+	NextBillingTime := timestamppb.New(c.NextBillingTime.Time())
+	if !c.NextBillingTime.IsSet() {
+		NextBillingTime = nil
+	}
+	BillingCycleStart := timestamppb.New(c.BillingCycleStart.Time())
+	if !c.BillingCycleStart.IsSet() {
+		BillingCycleStart = nil
+	}
```

Comment:
nit: would be good to introduce a method that does the conversion

Reply:
@svenefftinge Good idea. Where should this method live? Is it possible to add it to `go.Time` or `timestamppb` somehow?

Reply:
I would put it into /workspace/gitpod/components/usage/pkg/db/types.go. @easyCZ wdyt?

Reply:
The DB package shouldn't have any dependency on API level packages. If we need a conversion, it should be in the API package

Reply:
@easyCZ So in `usage.go` is fine? 😇

Reply:
I'd create `apiv1/time.go` or `apiv1/types.go` and put it there. There's nothing wrong with it being in `usage.go` but it could make it harder to discover that the helper exists. As per go guidance, avoiding files such as `util` or `helper` or `tools` is recommended to avoid kitchen-sink scenarios.

`types.go` is fairly standard and often used for the underlying types and conversions.

Reply:
Thanks for the guidance 👍

> As per go guidance, avoiding files such as `util` or `helper` or `tools` is recommended to avoid kitchen-sink scenarios.

💯 💯 💯 (love this about Go -- `util` is one of my pet peeves)